src/main/java/uk/gov/hmcts/juror/api/moj/domain/Juror.java

```diff
@@ -281,6 +281,10 @@ public class Juror extends Address implements Serializable {
     @NotAudited
     private String serviceCompCommsStatus;
 
+    @Column(name = "response_entered")
+    @NotAudited
+    private Boolean responseEntered;
```

Comment:
should be lower case (so defaults to false)

Reply:
updated